std_msgs/Header header
string[] record_topic_list
string split_duration_str
string save_name
string node_name
---
time record_time
bool success